paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -81,3 +108,9 @@ sealed class PersistablePaymentMethodOption(
         }
     }
 }
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class CustomerEphemeralKey(
```

Comment:
Should we make a factory method to create these? Should we keep the properties internal?